public/js/admin/inventory.js

```diff
@@ -22,6 +22,82 @@ $(document).ready(function () {
         columnDefs: [ { orderable: false, targets: [3] }]
     });
 
+    $(".currentQuantity").dblclick(function(){
+       $(this).find("span")[0].style.display="none";
+       $(this).find("input")[0].style.display="block";
+        $(this).find("input")[0].style.display="block";
+    });
+
+    // AJAX STUFF
+    var manageUsers = {
+        toggleUser: function (object) {
+            var id = $(object).attr('data-id');
+            var value = $(object).attr('value');
+            var tripId = $(object).attr('tripid');
+            $.ajax({
+                url: '/admin/inventory/edit/' + id + "/" + value + "/" + tripId,
+                type: 'POST',
+                data: {},
+                dataType: 'text',
```

Comment:
Typically when posting some data to a web server, the data to be updated (in this case value) should not be passed in via the url.

It is better to do something like:

```
$.ajax({
         url: '/admin/inventory/edit/' + id + "/" + tripId,
         type: 'POST',
        data: {
              'value': value
        },
```

Having the id and trip_id in the url is fine here because these values aren't being updated.